### PR TITLE
Wayland miscellany

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Motivation for this change

Reorder wayland tools

New packages

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/wayland/wl-clipboard/default.nix [hunk 1]
```diff
@@ -16,8 +23,8 @@ stdenv.mkDerivation rec {
   buildInputs = [ wayland ];
 
   meta = with lib; {
-    description = "Command-line copy/paste utilities for Wayland";
     homepage = "https://github.com/bugaevc/wl-clipboard";
+    description = "Command-line copy/paste utilities for Wayland";
     license = licenses.gpl3;
```
Comment (fabaff) [inline pkgs/tools/wayland/wl-clipboard/default.nix:28]: License seems to be [GPLv3+](https://github.com/bugaevc/wl-clipboard/blob/2c3cee1c8173d75b1411997b2c0f3e486194ac0c/src/util/misc.c#L5-L8).

```suggestion
    license = licenses.gpl3Plus;
```

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113319` at f529ec39 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/88614fa34ed5a7214de7175a99f91248">oguri</a></li></li>
    <li><a href="https://gist.github.com/5cfdd485ca7bccce61dfe77bd8edeeee">wlogout</a></li></li>
  </ul>
</details>

File: pkgs/tools/wayland/wl-clipboard/default.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/tools/wayland/wl-clipboard/default.nix:28]: I will add this as a new commit.

File: pkgs/tools/wayland/wl-clipboard/default.nix [hunk 2]
```diff
@@ -16,10 +23,11 @@ stdenv.mkDerivation rec {
...
-    description = "Command-line copy/paste utilities for Wayland";
     homepage = "https://github.com/bugaevc/wl-clipboard";
-    license = licenses.gpl3;
+    description = "Command-line copy/paste utilities for Wayland";
+    license = licenses.gpl3Plus;
     maintainers = with maintainers; [ dywedir ];
     platforms = platforms.linux;
   };
 }
+# TODO: is wayland-protocols a nativeBuildInput or a buildInput?
```
Comment (SuperSandro2000) [inline pkgs/tools/wayland/wl-clipboard/default.nix:33]: Is it a binary?

Comment (AndersonTorres) [reply pkgs/tools/wayland/wl-clipboard/default.nix:33]: It is a bunch of shell scripts and XML files:

https://github.com/wayland-project/wayland-protocols

**Pull request merged**